packages/flutter/test/material/chip_test.dart

```diff
@@ -93,10 +93,10 @@ Widget _wrapForChip({
 /// Optionally, adding an avatar or delete icon to the chip should not
 /// cause the chip or label to exceed its constrained height.
 Future<void> _testConstrainedLabel(
-  WidgetTester tester, {
-  CircleAvatar avatar,
-  VoidCallback onDeleted,
-}) async {
+    WidgetTester tester, {
```

Comment:
Looks like a bunch of formatting changes snuck in, can you please undo those

Reply:
Done

Reply:
Done